trade-finance-manager-api/api-tests/v1/deals/deals-submit.second-submission.api-test.js

```diff
@@ -84,6 +89,8 @@ describe('/v1/deals', () => {
     updatePortalFacilityStatusSpy.mockClear();
     externalApis.updatePortalFacilityStatus = updatePortalFacilityStatusSpy;
 
+    externalApis.updateGefMINActivity = updateGefActivitySpy;
```

Comment:
Best to do a `mockClear()` just before this - not an issue now but could trip someone up later on

Reply:
Could we add a check to make sure that when XYZ is submitted, `externalApis.updateGefMINActivity` is called?

`expect(updateGefActivitySpy).toHaveBeenCalledWith('....')`;

Reply:
done

Reply:
Can't see the test for spy being called, have you pushed?